components/modal/ModalLayout.tsx

```diff
@@ -0,0 +1,26 @@
+import Image from "next/image";
+import { ReactNode, MouseEvent } from "react";
+import close from "@/public/close.svg";
+import styles from "./ModalLayout.module.css";
+
+interface Props {
+  children: ReactNode;
+  title: string;
+  onClickCloseButton: (e: MouseEvent<HTMLDivElement>) => void;
+}
+
+function ModalLayout({ children, title, onClickCloseButton }: Props) {
+  return (
+    <div className={styles.background}>
+      <div className={styles.container}>
+        <div className={styles.modalTitle}>{title}</div>
+        <div className={styles.closeButton} onClick={onClickCloseButton}>
+          <Image src={close} alt="Icon-close" />
```

Comment:
이미지의 alt 속성값은 해당 이미지가 출력되지 않고 텍스트로 대체된다고 가정했을 때, 쉽게 이해할 수 있게 간단명료하게 작성하시면 됩니다. 네이밍 컨벤션을 맞추지 않아도 되고요!
```suggestion
          <Image src={close} alt="닫기" />
```

Reply:
리뷰 남기지 않은 다른 부분도 확인해 주세요!
`'이미지', '아이콘'` 과 같은 키워드는 들어가지 않아도 됩니다.